airbyte_cdk/sources/declarative/models/declarative_component_schema.py

```diff
@@ -1720,18 +1701,13 @@ class Config:
         extra = Extra.allow
 
     type: Literal["ZipfileDecoder"]
-    parser: Union[GzipParser, JsonParser, JsonLineParser, CsvParser] = Field(
+    decoder: Union[CsvDecoder, GzipDecoder, JsonDecoder, JsonlDecoder] = Field(
```

Comment:
Asking to learn: why is it that `GzipDecoder` has `innner_decoder`, but `ZipfileDecoder` has just `decoder`? Should these names be consistent?

Reply:
We should probably have those name consistent. As we are currently doing the breaking change, I can probably break this as well